.github/workflows/publish-pypi.yml

```diff
@@ -0,0 +1,35 @@
+# This workflow will upload a Python Package using Twine when a release is created
+# For more information see: https://help.github.com/en/actions/language-and-framework-guides/using-python-with-github-actions#publishing-to-package-registries
+
+name: Upload Python Package to Test PyPI
+
+on:
+  release:
+    types: [created]
+
+jobs:
+  deploy:
+
+    runs-on: ubuntu-latest
+
+    steps:
+    - name: Check out mitiq
+      uses: actions/checkout@v2
+      with:
+        ref: 'release'
+    - name: Set up Python
+      uses: actions/setup-python@v2
+      with:
+        python-version: 3.8
+    - name: Install Python dependencies
+      run: |
+        python -m pip install --upgrade pip
+        make install requirements
+        pip install setuptools wheel twine
+    - name: Build and publish
+      env:
+        TWINE_USERNAME: ${{ secrets.PYPI_USERNAME }}
+        TWINE_PASSWORD: ${{ secrets.PYPI_PASSWORD }}
```

Comment:
I understand this is connected with @crazy4pi314 account, with secrets added to the repo. I think it would be better to create an additional maintainer for this See e.g., bot. https://pypi.org/project/cirq/, https://pypi.org/project/tf-nightly/.

Reply:
I have updated the repo secrets to use a bot account (UnitaryFundBot) for both PyPI and TestPyPI workflows, does this work?